### DISCUSSION TITLE
How to change the title at table of relation manager ?

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5282] [Help]
Hi All,

How to change the title at table of relation manager ?

![image](https://user-images.githubusercontent.com/43429317/208287820-6af9f90c-e7ee-4686-9465-5e0257f54593.png)

Thanks

**Comment by danharrin** [comment]
Override `getTableHeading()`

**Comment by altimusin** [comment]
Thank you..

But may I know where I can found the documentation of getTableHeading().
So I can find-out and learn the other functionality of a table.

Thank You.
